speakeasy/windows/win32.py

```diff
@@ -240,7 +240,7 @@ def load_module(self, path=None, data=None, first_time_setup=True):
         self.setup(first_time_setup=first_time_setup)
 
         if not self.stack_base:
-            self.stack_base, stack_addr = self.alloc_stack(0x12000)
+            self.stack_base, stack_addr = self.alloc_stack(pe.OPTIONAL_HEADER.SizeOfStackReserve)
```

Comment:
```suggestion
            self.stack_base, stack_addr = self.alloc_stack(pe.OPTIONAL_HEADER.SizeOfStackReserve or 0x12000)
```

add guard in the case where this field might be zero. not sure if thats allowable by the PE loader but we should be careful.

Reply:
ok @cecio ?

Reply:
makes totally sense, yes!